doc/HttpLuaModule.wiki

```diff
@@ -4686,12 +4686,20 @@ This method was introduced in the <code>0.5.0rc30</code> release.
 
 == ngx.escape_uri ==
 
-'''syntax:''' ''newstr = ngx.escape_uri(str)''
+'''syntax:''' ''newstr = ngx.escape_uri(str, type?)''
 
 '''context:''' ''init_by_lua*, init_worker_by_lua*, set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, body_filter_by_lua*, log_by_lua*, ngx.timer.*, balancer_by_lua*, ssl_certificate_by_lua*, ssl_session_fetch_by_lua*, ssl_session_store_by_lua*''
 
 Escape <code>str</code> as a URI component.
 
+Since `v0.10.16rc6`, this function accepts an optional <code>type</code> argument.
+<geshi lang="lua">
+  type = 0 (Escape <code>str</code> as a URI)
+  type = 2 (Escape <code>str</code> as a URI component)
+</geshi>
+when type set to 0, " ", "#", "%", "?", 0x00-0x1F, 0x7F-0xFF will be escaped.
+when type set to 2, all characters excepter ALPHA, DIGIT, "-", ".", "_", "~" will be escaped.
```

Comment:
You should combine these two lists into a single list. Each case is explained twice above.